### DISCUSSION TITLE
Logs are not showing detail in production!

### DISCUSSION THREAD

**Discussion by teddis** [vercel/community#3655] [Help] [answered]
### Summary

It looks like my hobby project is forced into production mode? I can't see the errors produced when I click on the error log item. How do I rectify this ASAP!?

### Example

https://vercel.com/teddis/sotalmvp/5EnfQtuBzP6YshB4FnENRCSrCHrw/logs?page=1&timeline=maximum&startDate=1692300376780&endDate=1692303976780&levels=error

### Steps to Reproduce

1. Run my instance at http://mvp.sotalented.io/
2. Sign in via Google SSO at bottom right
3. Get a 504 error, but there's no detail to track down this error.
This is urgent. Thanks.

**Comment by amyegan** [comment, marked as answer]
Hey @teddis. A [500: FUNCTION_INVOCATION_FAILED](https://vercel.com/docs/errors/application-error-codes#500:-function_invocation_failed) error is most often caused by an unhandled exception in the code. 

Please double check that all necessary environment variables are correct and valid for the deployment environment in project settings. This is a common reason why external service calls fail.

I also recommend wrapping sections of code in [try...catch statements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) which allow you to log extra information for debugging.
